src/autolabel/tasks/multi_label_classification.py

```diff
@@ -0,0 +1,200 @@
+from collections import defaultdict
+from typing import List, Dict, Tuple
+
+from langchain.prompts.prompt import PromptTemplate
+from sklearn.metrics import accuracy_score
+
+from autolabel.confidence import ConfidenceCalculator
+from autolabel.configs import AutolabelConfig
+from autolabel.schema import LLMAnnotation, Metric, MetricResult
+from autolabel.tasks import BaseTask
+from autolabel.tasks.utils import compute_f1
+from autolabel.utils import get_format_variables
+
+import json
+
+
+class MultiLabelClassificationTask(BaseTask):
+    DEFAULT_OUTPUT_GUIDELINES = 'You will return the answer as a comma separated list of labels. For example: "label1, label2, label3"'
```

Comment:
how well does this guideline do across LLMs ? A few other choices possible here are:
- return it separated by newlines
- return it as a list/array

I'm neutral to what we choose ultimately, but whatever is the default output guideline should work well across LLMs (including Refuel LLM)